Solutions/Infoblox NIOS/Data Connectors/Connector_Syslog_Infoblox.json

```diff
@@ -8,30 +8,30 @@
      { 
          "metricName": "Total data received", 
          "legend": "InfobloxNIOS", 
-         "baseQuery": "Infoblox_all" 
+         "baseQuery": "Infoblox_consolidated" 
      } 
     ], 
         "sampleQueries": [ 
         { 
-            "description" : "Total Count by DHCP Message Types", 
-            "query": "Infoblox_dhcp_consolidated \n| where LogType == \"dhcpd\"  \n| where Type startswith \"DHCP\" \n| summarize count() by Type"    
+            "description" : "Total Count by DHCP Request Message Types", 
+            "query": "union isfuzzy=true \n Infoblox_dhcpdiscover,Infoblox_dhcprequest,Infoblox_dhcpinform \n| summarize count() by Log_Type"    
         },
         {
             "description" : "Top 5 Source IP address",
-            "query": "Infoblox_dns_consolidated \n | where LogType == \"named\" and Type == \"client\" \n| summarize count() by Client_IP \n | top 10 by count_ desc" 
+            "query": "Infoblox_dnsclient \n | summarize count() by Client_IP \n | top 10 by count_ desc" 
         }
         ], 
             "dataTypes": [ 
             { 
                "name": "Syslog (InfobloxNIOS)", 
-               "lastDataReceivedQuery": "Infoblox_all \n            | summarize Time = max(TimeGenerated)\n            | where isnotempty(Time)" 
+               "lastDataReceivedQuery": "Infoblox_consolidated \n            | summarize Time = max(TimeGenerated)\n            | where isnotempty(Time)" 
             } 
         ], 
              "connectivityCriterias": [ 
                 { 
                     "type": "IsConnectedQuery", 
                     "value": [ 
-                    "Infoblox_all \n      | summarize LastLogReceived = max(TimeGenerated)\n            | project IsConnected = LastLogReceived > ago(7d)" 
+                    "Infoblox_consolidated \n | where TimeGenerated > ago(3d)\n |take 1 \n | project isConnected = true" 
```

Comment:
@ms-sapangoel : are we standardizing on 3 days as the connection testing period? It is important to be consistent here.

Reply:
I had a discussion with Sapan on this and we are consistent with 3 days as the connection testing period.